content/en/docs/concepts/cluster-administration/swap-memory-management.md

```diff
@@ -0,0 +1,397 @@
+---
+title: Swap memory management
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Kubernetes can be configured to use swap memory on a {{< glossary_tooltip text="node" term_id="node" >}}
+to increase available memory for workloads.
```

Comment:
I don't know if I would say that swap memory increases the available memory.

Its more of a backing store in case of kernel memory pressure. We also don't increase the allocatable memory if there is swap. A node will still have the same amount of memory with/without swap in this case.

Reply:
You're right! This phrasing is not accurate.
I've replaced it with:

> Kubernetes can be configured to use swap memory on a {{< glossary_tooltip text="node" term_id="node" >}},
allowing the kernel to free up physical memory by swapping out pages to backing storage.

WDYT?